project/profiles/capacity4more/modules/c4m/content/c4m_content/c4m_content.module

```diff
@@ -37,6 +37,22 @@ function c4m_content_form_node_form_alter(&$form, &$form_state, $form_id) {
       '#weight' => 14,
     );
   }
+
+  // Removes 'draft' form action (e.g. Unpublish button)
+  unset($form['actions']['draft']);
```

Comment:
@guytepper 
Please add a condition to make sure that the `#value` on the draft is `t('Unpublished')`